assets/js/print.js

```diff
@@ -0,0 +1,7 @@
+function showDetailsElementOnPrint() {
+  document.querySelectorAll('.govuk-details').forEach(details => {
+    details.setAttribute('open', '')
+  })
+}
+
+window.addEventListener('beforeprint', () => showDetailsElementOnPrint())
```

Comment:
We need to use JavaScript here, as we cannot use CSS to expand details elements as its currently not in the CSS spec.

See: https://github.com/w3c/csswg-drafts/issues/2084

Reply:
So there's a possible (albeit remote) accessibility issue here in that if the browser does not have JS enabled (I accept this is unlikely) then they wont be able to print the notes as part of the goals on the overview page?

Which leads me onto ..... I thought there was discussion/conversation about whether the notes should be printed at all? The last I heard was No as there could be sensitive into in the notes that might not be appropriate to be printed/left lying around/given to the prisoner ? , but perhaps that has changed since then?